### ISSUE TITLE
padj and log2FC

### ISSUE THREAD

**Issue by hamzaaitabbou96** [mzytnicki/srnadiff#2]
Hi,

I have a question how can i extract the count information which we used to calculate padj and log2FC ?

**Comment by mzytnicki** [comment]
You can access the count matrix with `countMatrix(srnaExp)`, supposing that `srnaExp` is your object.

Does that answer your question?

**Comment by hamzaaitabbou96** [comment]
i have this results 
![res](https://github.com/mzytnicki/srnadiff/assets/65475732/57a579f9-da1b-412d-9afd-572e4efafdef)

i want to know the number of reads for each miRNA which we used to calculate padj and log2FC

**Comment by mzytnicki** [comment]
Thanks.  Did you try the method `countMatrix`?  What is the output?

**Comment by hamzaaitabbou96** [comment]
yes, the output is :

![res2](https://github.com/mzytnicki/srnadiff/assets/65475732/b229a7b8-5726-4b55-8931-f2b585d4c85f)

**Comment by mzytnicki** [comment]
I see.  Please leave me some time to install the fresh version of R (4.3.0)...

**Comment by hamzaaitabbou96** [comment]
thanks, i'm waiting for your answer

**Comment by mzytnicki** [comment]
Now I am back!  For some reason, I could not install `baySeq`... So, currently, there is no possibility for you to get the raw counts from the output...  I could try to implement it, if you wish.  But you would need to be very patient: I have several urgent tasks to do...  Still interested?

**Comment by mzytnicki** [comment]
Well, actually, I did a quick fix.
What you can do is:

    install.packages("devtools")
    devtools::install_github("mzytnicki/srnadiff", ref = "region_names")

You can then restart the pipe-line.  The count matrix should now reflect the region names.  Does that work?

**Comment by hamzaaitabbou96** [comment]
I did the command but i got this error ERROR: dependency 'baySeq' is not available for package 'srnadiff'

about the update that you did, you replace length(intervals) by seqnames(intervals)_start(intervals)_end(intervals)
for example in this file IR_1 by chr1_30438_30458

![res2](https://github.com/mzytnicki/srnadiff/assets/65475732/d3048177-5dfb-481b-b841-2fc612267118)

**Comment by mzytnicki** [comment]
Yes, it seems that there is a problem for `baySeq`, but I do not know what is going wrong.
Now that you have the count matrix with a usable column name, you can now make a join with the result table.
Does that help?  Do you know how to do it?
